WebRoot/skins/_base/base4/skin.properties

```diff
@@ -457,11 +457,12 @@ ZFieldBg-disabled           = background-color:transparent;
 #####################
 #   Inputs: input type=[text|password] elements only
 #####################
-InputBorder                 = border:1px solid; border-color:@darken(AppC,75)@ @darken(AppC,10)@ @darken(AppC,10)@ @darken(AppC,75)@;
+InputBorder                 = border:1px solid #CCC;
+InputHeight                 = height:32px; height:2.65rem;
```

Comment:
this property is used once only, so we can move it inline
or are you thinking to make this a generic property which will be used by all input fields?